io.itsterminal.WebPConverter.json

```diff
@@ -0,0 +1,35 @@
+{
+  "app-id": "io.itsterminal.WebPConverter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "webp_converter.py",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
```

Comment:
why is home needed?

Reply:
this i think is a something i had before i learnt about flatpak portals. Will remove.

Reply:
removed.